sim/paladin/retribution/retribution_test.go

```diff
@@ -49,6 +49,7 @@ func TestRetribution(t *testing.T) {
 		},
 		{
 			Class:      proto.Class_ClassPaladin,
+			Phase:      4,
```

Comment:
This is the phase 3 set right? Looks like it's mis-labeled but you can omit phase for 25/40/50 and it will automatically figure it out

Reply:
fixed and repushed